src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculator.java

```diff
@@ -123,27 +124,27 @@ public void setGameData(final GameData data) {
   public void setCalculateData(final PlayerID attacker, final PlayerID defender, final Territory location,
       final Collection<Unit> attacking, final Collection<Unit> defending, final Collection<Unit> bombarding,
       final Collection<TerritoryEffect> territoryEffects, final int runCount) throws IllegalStateException {
-    if (m_isRunning) {
+    if (isRunning) {
       return;
     }
-    m_isCalcSet = false;
-    if (!m_isDataSet) {
+    isCalcSet = false;
+    if (!isDataSet) {
       throw new IllegalStateException("Called set calculation before setting game data!");
     }
-    m_attacker =
-        m_data.getPlayerList().getPlayerID((attacker == null ? PlayerID.NULL_PLAYERID.getName() : attacker.getName()));
-    m_defender =
-        m_data.getPlayerList().getPlayerID((defender == null ? PlayerID.NULL_PLAYERID.getName() : defender.getName()));
```

Comment:
Unnecessary parentheses removed from lines 134 and 136 to avoid a Checkstyle line length violation after renaming `m_data` to `gameData`.